tracer/build/_build/_build.csproj

```
@@ -58,13 +58,18 @@
     <Content Include="..\..\..\.azure-pipelines\ultimate-pipeline.yml" Link="ci\ultimate-pipeline.yml" />
     <Content Include="..\..\build_in_docker.sh" Link="boot\build_in_docker.sh" />
     <Content Include="..\..\build_in_docker.ps1" Link="boot\build_in_docker.ps1" />
+    <Content Include="..\..\..\.github\CODEOWNERS" Condition="Exists('..\..\..\.github\CODEOWNERS')" Link="CodeOwners\CODEOWNERS">
+      <CopyToOutputDirectory>Always</CopyToOutputDirectory>
```

_Generally_ using `Always` slows builds. Theoretically.
```suggestion
      <CopyToOutputDirectory>PreserveNewest</CopyToOutputDirectory>
```

---

Thanks!